clients/js/test/plugins/asset/addBlocker.test.ts

```diff
@@ -0,0 +1,123 @@
+import test from 'ava';
+
+import { addPluginV1, createPlugin, pluginAuthorityPair } from '../../../src';
+import {
+  DEFAULT_ASSET,
+  assertAsset,
+  createAsset,
+  createUmi,
+} from '../../_setup';
+
+test('it cannot add UA-managed plugin if addBlocker had been added on creation', async (t) => {
```

Comment:
Can you add tests asserting that the Update Authority (as a separate generated keypair) is the only one who can add this plugin. Owners can't, random keypairs can't, UA can.